net/strongswan/files/ipsec.init

```diff
@@ -1,20 +1,327 @@
 #!/bin/sh /etc/rc.common
 
-START=90
-STOP=10
+NAME=ipsec
+START=21
+STOP=60
+USE_PROCD=1
```

Comment:
If the previous script wasn't using procd, I prefer the switch to procd in a different commit. Make sure to check https://wiki.openwrt.org/inbox/procd-init-scripts